stdlib/multiprocessing/__init__.pyi

```diff
@@ -62,27 +60,6 @@ __all__ = [
 if sys.version_info >= (3, 8):
     __all__ += ["parent_process"]
 
-# The following type aliases can be used to annotate the return values of
-# the corresponding functions. They are not defined at runtime.
```

Comment:
These were a bit controversial when @srittau added them, and he probably doesn't want them removed: #5346

Reply:
Ah, thanks for the reference.

Reply:
The other options are:

- Special-case `__init__.pyi` files in the flake8-pyi check (what I did originally, but @JelleZijlstra doesn't like that).
- Just `# noqa` all the ones in `multiprocessing`.
- Abandon the flake8-pyi check entirely.
- Something else?

Reply:
I think this is somewhat of a "true positive". These type aliases are just weird: named as if they were private, intended to be public, not documented, not discoverable. But again, @srittau will probably disagree. I think we should wait for his input.

Reply:
Yeah of course, not going to merge without it :)